src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/EntityPublished/oidcConfirmationModal.html.twig

```diff
@@ -3,8 +3,8 @@
 {% block page_heading %} {{ 'entity.list.modal.oidc_confirmation.title'|trans }} {% endblock %}
 
 {% block body %}
-    <div class="warning"><i class="fa fa-exclamation-triangle"></i>{{ 'entity.list.modal.oidc_confirmation.warning'|trans|raw }}</div>
-    <div class="wysiwyg">{{ 'entity.list.modal.oidc_confirmation.info.html'|trans|raw }}</div>
+    <div class="warning"><i class="fa fa-exclamation-triangle"></i>{{ 'entity.list.modal.oidc_confirmation.warning'|trans|wysiwyg }}</div>
```

Comment:
I don't think filtering the output is warranted here. The original input is not filled using the wysiwyg editor.

Reply:
Was indeed not necessary so removed it

Reply:
FYI: `entity.list.modal.oidc_confirmation.info.html` should have the wysiwyg filtering as this can be loaded with HTML content. I was commenting on the `entity.list.modal.oidc_confirmation.warning` translation. Which is a pure text translation that does not benefit from html output sanitisation.

Reply:
I removed the wrong filter, fixed